Makefile

```diff
@@ -138,6 +138,7 @@ go-install-cli:
 	$(GOINSTALL) -trimpath github.com/lightninglabs/faraday/cmd/frcli
 	$(GOINSTALL) -trimpath -ldflags "$(LDFLAGS)" github.com/lightninglabs/pool/cmd/pool
 	$(GOINSTALL) -trimpath -ldflags "$(LDFLAGS)" github.com/lightninglabs/taproot-assets/cmd/tapcli
+	$(GOINSTALL) -tags="$(LND_RELEASE_TAGS)" -ldflags "$(LDFLAGS)" $(PKG)/cmd/litcli
```

Comment:
I was a bit unsure if I should include the `-trimpath` argument here, as we're only adding the cli executable. Though I figured that users will likely run both `make go-install `and `make go-install-cli`. I therefore opted to not do so, as I didn't want that the order the commands are run in to effect the resulting `litcli` executable after running both commands.

Reply:
The `-trimpath` removes local directory paths to golang modules and replaces them with static ones, helping with the binary reproducibility of the release binaries. So IMO we should add the `-trimpath` flag to every binary that is not a debug/dev build. Meaning that we should probably also have them in `go-install`.

Reply:
Ok, thanks for the feedback! Updated by adding the `trimpath` flag to both this, and `go-install`!